Dockerfile.dapper

```diff
@@ -32,7 +32,7 @@ RUN curl -sL ${HELM_URL} | tar -xzf - && \
     HOME=/go/src/github.com/rancher/externalip-webhook/ CATTLE_HELM_UNITTEST_VERSION=${HELM_UNITTEST_VERSION} /usr/bin/helm plugin install ${HELM_UNITTEST_URL} \
```

Comment:
Does chart and app version just depend on the tag ?

Reply:
yes both depends on the tag